src/richie/apps/courses/models/enrollment.py

```diff
@@ -0,0 +1,70 @@
+"""
+Declare and configure `courses` application models related to enrollment.
+"""
+from django.contrib.auth import get_user_model
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from .course import CourseRun
+
+
+class Enrollment(models.Model):
+    """
+    Enrollment model. Links a user to the course runs they have enrolled in.
+
+    In most cases, actual enrollment also needs to happen in the actual LMS behind Richie.
+    Still, we need a simple and fast way to look up and manage user enrollments in
+    Richie itself, which is tough to achieve if every action necessitates a bunch of
+    API calls to different LMSes.
```

Comment:
Reading this made me realize that we need a feature switch on this so that it is still possible to use Richie without integration with LMS. In this mode, the course run just provides a link to the course on the LMS.
This will also allow us to merge this PR without breaking things until the user dashboard and connection backend are also available?

Reply:
Good call, I had not thought about this 👍

Reply:
Should this be a separate app then? So if you don't use Richie enrollments you don't even have to create db tables and everything?

Reply:
Yes it's probably a good idea. And the whole user dashboard, ecommerce etc. will be in this app... In any case, make sure to force your db table names so that we can move things around if the choice we make turns out to be wrong.